app/helpers/react_components/settings/theme_preference_form.rb

```diff
@@ -0,0 +1,21 @@
+module ReactComponents
+  module Settings
+    class ThemePreferenceForm < ReactComponent
+      def to_s
+        super("settings-theme-preference-form", {
+          default_theme_preference:,
+          insiders_status: current_user.insiders_status,
+          links: {
+            # TODO: add this field too, check this update URL
+            update: Exercism::Routes.api_settings_url
+          }
+        })
+      end
+
+      # TODO: add these as default values in DB
+      def default_theme_preference
+        %i[eligible eligible_lifetime].include?(current_user.insiders_status) ? 'system' : 'light'
```

Comment:
This should be `active` variants, not `eligible` variants. Check this in all your PRs pls